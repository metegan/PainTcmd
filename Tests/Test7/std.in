PL Name12 56 46 108 4536 80 100
L 1172v 56 46 108 4536
R rectangle1 56 46 108 4536
OA agg1 rectangle1 1172v
MOVE Name12 100 -25
MOVE agg1 1000 1000
EXIT

