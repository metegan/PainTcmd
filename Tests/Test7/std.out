OK
# New object : Name12
OK
# New object : 1172v
OK
# New object : rectangle1
OK
# New object : agg1
OK
# Object Name12 moved
OK
# Object agg1 moved
